frontend/src/app/shared/components/blog-preview-vertical/blog-preview-vertical.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="blog-vertical">
+  <div class="image-blog"></div>
+  <div class="bottom">
+    <div class="bottom-tag"><cvz-tag class="narrow">Vegetable</cvz-tag></div>
+    <div class="text">Which vegetable your family will love and want’s eat each day</div>
+    <div class="footer">
+      <div class="author">Author</div>
+      <div class="date">12.12.2012</div>
+    </div>
+  </div>
```

Comment:
as scoate clasa bottom, as prefera sa nu puneti la nume de clase top, left, bottom right, doar asa simple.


iti sugerez sa folosesti in cazul tau, 

`<section>` in loc de `<div class="bottom">` si are rol semantic mai ok.

iar `<cvz-tag>` nu cred ca mai are nevoie de div-ul asta pe exterior `<div class="bottom-tag">`